src/common/utility.ts

```diff
@@ -343,6 +344,51 @@ export class Utility {
         } catch (error) {}
     }
 
+    public static getAddressKey(address: string, keySet: Set<string>): string {
+        let key = address;
+        let index = address.indexOf(".");
+        if (index === -1) {
+            index = address.indexOf(":");
+        }
+        if (index !== -1) {
+            key = address.substring(0, index);
+        }
+
+        let suffix = 1;
+        const keyPrefix: string = key;
+        while (keySet.has(key)) {
+            key = `${keyPrefix}_${suffix}`;
+            suffix += 1;
+        }
+
+        return key;
+    }
+
+    public static getRegistryAddress(repositoryName: string) {
+        const DefaultHostname = "docker.io";

```

Comment:
`DefaultHostname` -> `defaultHostname`, `LegacyDefaultHostname` -> `legacyDefaultHostname` to keep naming style consistent

Reply:
Thx, will update